Products/ZenModel/migrate/addOpenTSDBLogbackConfig.py

```diff
@@ -0,0 +1,65 @@
+        log.info("Of those, %i services use /opt/zenoss/etc/opentsdb/opentsdb.conf." % len(services))
+        hm_content = '<?xml version="1.0" encoding="UTF-8"?>\n<configuration>\n  <!--<jmxConfigurator/>-->\n  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">\n    <encoder>\n      <pattern>\n        %d{ISO8601} %-5level [%thread] %logger{0}: %msg%n\n      </pattern>\n    </encoder>\n  </appender>\n\n  <!-- This appender is responsible for the /logs endpoint. It maintains MaxSize\n       lines of the log file in memory. If you don\'t need the endpoint, disable\n       this appender (by removing the line "<appender-ref ref="CYCLIC"/>" in\n       the "root" section below) to save some cycles and memory. -->\n  <appender name="CYCLIC" class="ch.qos.logback.core.read.CyclicBufferAppender">\n    <MaxSize>1024</MaxSize>\n  </appender>\n\n  <!-- Appender to write OpenTSDB data to a set of rotating log files -->\n  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">\n    <file>/var/log/opentsdb/opentsdb.log</file>\n    <append>true</append>\n\n    <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">\n      <fileNamePattern>/var/log/opentsdb/opentsdb.log.%i</fileNamePattern>\n      <minIndex>1</minIndex>\n      <maxIndex>3</maxIndex>\n    </rollingPolicy>\n\n    <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">\n      <maxFileSize>128MB</maxFileSize>\n    </triggeringPolicy>\n\n    <encoder>\n      <pattern>%d{HH:mm:ss.SSS} %-5level [%logger{0}.%M] - %msg%n</pattern>\n    </encoder>\n  </appender>\n\n  <!-- Appender for writing full and completed queries to a log file. To use it, make\n       sure to set the "level" to "INFO" in QueryLog below. -->\n  <appender name="QUERY_LOG" class="ch.qos.logback.core.rolling.RollingFileAppender">\n    <file>/var/log/opentsdb/queries.log</file>\n    <append>true</append>\n\n    <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">\n        <fileNamePattern>/var/log/opentsdb/queries.log.%i</fileNamePattern>\n        <minIndex>1</minIndex>\n        <maxIndex>4</maxIndex>\n    </rollingPolicy>\n\n    <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">\n        <maxFileSize>128MB</maxFileSize>\n    </triggeringPolicy>\n    <encoder>\n        <pattern>%date{ISO8601} [%logger.%M] %msg%n</pattern>\n    </encoder>\n  </appender>\n\n  <!-- Per class logger levels -->\n  <logger name="QueryLog" level="OFF" additivity="false">\n    <appender-ref ref="QUERY_LOG"/>\n  </logger>\n  <logger name="org.apache.zookeeper" level="INFO"/>\n  <logger name="org.hbase.async" level="WARN"/>\n  <logger name="com.stumbleupon.async" level="WARN"/>\n\n  <!-- Fallthrough root logger and router -->\n  <root level="INFO">\n    <appender-ref ref="STDOUT"/>\n    <appender-ref ref="CYCLIC"/>\n    <appender-ref ref="FILE"/>\n  </root>\n</configuration>\n\n'
```

Comment:
It's hard to tell what is in the this inline XML string or if it matches the value in the corresponding service definition in order to verify that it's correct. Please move this content to a file like `Products/ZenModel/migrate/data/opentsdb-logback.xml` and then have the script read that file at runtime.